website/src/react-native/showcase.js

```diff
@@ -327,7 +327,16 @@ var featured = [
     linkPlayStore: 'https://play.google.com/store/apps/details?id=com.deliverycom&hl=en',
     infoLink: 'https://medium.com/delivery-com-engineering/react-native-in-an-existing-ios-app-delivered-874ba95a3c52#.37qruw6ck',
     infoTitle: 'React Native in an Existing iOS App: Getting Started'
-  },  
+  },
+  {
+    name: 'adidas GLITCH',
+    icon: 'http://d2w4pyf30enukh.cloudfront.net/prod/glitch-logo.jpg/tnm.jpg',
+    linkAppStore: 'https://itunes.apple.com/gb/app/adidas-glitch-interchangeable/id1166181522?mt=8&cm_sp=FOOTBALL-_-UNDEFINED-_-CTA-DOWNLOAD-IPHONE',
+    linkPlayStore: 'https://play.google.com/store/apps/details?id=com.adidas.glitch&cm_sp=FOOTBALL-_-UNDEFINED-_-CTA-DOWNLOAD-ANDROID',
+    infoLink: 'http://www.campaignlive.co.uk/article/adidas-launches-glitch-boot-invite-only-app-revolutionary-approach/1416060',
+    infoTitle: 'GLITCH - Break the pattern',
+    defaultLink: 'http://www.campaignlive.co.uk/article/adidas-launches-glitch-boot-invite-only-app-revolutionary-approach/1416060',
```

Comment:
You don't need a defaultLink here, as this is only used for apps in the pinned array.